gplately/download.py

```diff
@@ -444,13 +461,92 @@ def get_rasters(self, raster_id_string=None, time=None):
+            "Muller2019_jpeg" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2019_Tectonics/Muller_etal_2019_Agegrids/Muller_etal_2019_Tectonics_v2.0_jpgs.zip"],
+            "Muller2019_png" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2019_Tectonics/Muller_etal_2019_Agegrids/Muller_etal_2019_Tectonics_v2.0_pngs.zip"],
+            "Muller2016_netCDF" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2016_AREPS/Muller_etal_2016_AREPS_Agegrids/Muller_etal_2016_AREPS_Agegrids_v1.17/Muller_etal_2016_AREPS_v1.17_netCDF.zip"],
+            "Muller2016_jpeg" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2016_AREPS/Muller_etal_2016_AREPS_Agegrids/Muller_etal_2016_AREPS_Agegrids_v1.17/Muller_etal_2016_AREPS_v1.17_jpgs.zip"],
+            "Muller2016_png" : ["https://www.earthbyte.org/webdav/ftp/Data_Collections/Muller_etal_2016_AREPS/Muller_etal_2016_AREPS_Agegrids/Muller_etal_2016_AREPS_Agegrids_v1.17/Muller_etal_2016_AREPS_v1.17_pngs.zip"],
+        }
+
+        archive_formats = tuple([".gz", ".xz", ".bz2"])
+        # Set to true if we find the given collection in database
+        found_collection = False
+        raster_filenames = []
+        for collection, zip_url in database.items():
+            # Isolate the plate model and the file type
+            plate_model = collection.split("_")[0]
+            raster_type = collection.split("_")[-1]
+            if (self.file_collection.lower() == plate_model.lower()
+                and filetype.lower() == raster_type.lower()
+                ):
+                found_collection = True
+                raster_filenames = _order_filenames_by_time(
+                    _collect_file_extension(
+                        _fetch_from_web(zip_url[0]), _match_filetype_to_extension(filetype))
+                )
+                
+                if time is not None:
+                    raster_filenames = _order_filenames_by_time(raster_filenames)[time]
+                break
+
+        if found_collection is False:
+            raise ValueError("%s not in collection database." % (raster_id_string))
+        return raster_filenames
+
+
+    def get_raster(self, raster_id_string=None, filetype=None):
+        """Downloads assorted rasters and images from the web into the "gplately" cache.
+
+        Currently supports the following rasters and images:
+        +--------------+------------------------+---------------------------------------+-------------------+
+        | Raster/image | Description            | Citation                              | String identifier |
+        | name         |                        |                                       |                   |
+        +--------------+------------------------+---------------------------------------+-------------------+
         | ETOPO1       | A 1-arc minute global  | doi:10.7289/V5C8276M                  | "ETOPO1_grd"      |
```

Comment:
I think this needs updating - the `raster_id_string` can be `ETOPO1`. It doesn't matter what the extension is since the user is returned a masked numpy array.